dev

```diff
@@ -12,10 +12,10 @@ MESG_LOG_FORCECOLORS=${MESG_LOG_FORCECOLORS:-false}
 MESG_LOG_LEVEL=${MESG_LOG_LEVEL:-debug}
 
 VERSION=local
-LDFLAGS="-X 'github.com/mesg-foundation/core/version.Version=$VERSION'"
-LDFLAGS+=" -X 'github.com/mesg-foundation/core/config.EnvMarketplaceEndpoint=https://ropsten.infura.io/v3/6115ae2531f64c04a9a392cf500e5fbe'"
-LDFLAGS+=" -X 'github.com/mesg-foundation/core/config.EnvMarketplaceAddress=0xeCC1A867F871323350A1A89FcAf69629a2d5085e'"
-LDFLAGS+=" -X 'github.com/mesg-foundation/core/config.EnvMarketplaceToken=0x5861b3dc52339d4f976b7fa5d80db6cd6f477f1b'"
+LDFLAGS="-X 'github.com/mesg-foundation/engine/version.Version=$VERSION'"
```

Comment:
use core here, this will probably not work

Reply:
how it won't work? can you  explain?